### PR TITLE
science/orthanc-postgresql: Revive port

### PR DESCRIPTION
**Pull request by yzgyyang**
Orthanc plugin to use PostgreSQL for indexing or storage.

PR:		242548

### CODE REVIEW

File: science/orthanc-postgresql/files/patch-PostgreSQL_CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--- PostgreSQL/CMakeLists.txt.orig	2021-05-17 12:24:06 UTC
++++ PostgreSQL/CMakeLists.txt
+@@ -23,6 +23,9 @@ set(OPENSSL_STATIC_VERSION "1.1.1" CACHE STRING "Force
+ 
+ include(${CMAKE_SOURCE_DIR}/../Resources/CMake/DatabasesPluginConfiguration.cmake)
+ 
++set(CMAKE_CXX_STANDARD 11)
```
Comment (lwhsu) [inline science/orthanc-postgresql/files/patch-PostgreSQL_CMakeLists.txt:7]: Does it mean that we need `USES=compiler:c11` ?

File: science/orthanc-postgresql/pkg-descr [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Orthanc can be extended with two plugins that replace the default SQLite
```
Comment (lwhsu) [inline science/orthanc-postgresql/pkg-descr]: two plugins?

File: science/Makefile [hunk 3]
```diff
@@ -177,7 +177,11 @@
     SUBDIR += opensph
     SUBDIR += opsin
     SUBDIR += orthanc
+<<<<<<< HEAD
     SUBDIR += orthanc-webviewer
+=======
+    SUBDIR += orthanc-postgresql
+>>>>>>> 21b9f5c6fea6 (science/orthanc-postgresql: Revive port)
```
Comment (lwhsu) [inline science/Makefile]: unresolved conflict.

Comment (yzgyyang) [reply science/Makefile]: Sorry - this port should still be wip, and I accidentally tagged you.

**Pull request merged**